charts/jupyterhub/templates/hub/rbac.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.rbac.enabled -}}
```

Comment:
This is going to need to be retooled for cluster roles and creating ancillary objects.

We should think about whether the "additional-K8s-stuff" service is going to run as the Hub user (and just pile all those privileges into it) or as a separate user, in which case picking the RBAC permissions apart will be trickier.

Reply:
Doesn't have to happen immediately, but will as soon as we want user namespaces.

Reply:
Well I think we want the hub part to be creating k8s stuff just as YAML and running it through, I think having it as the hub user makes sense because I'm not sure what else the hub user is really doing in this context.  But yeah we'll need a lot of work to get user namespaces to work, because a lot of things depend in subtle ways on the stuff in the namespace